### ISSUE TITLE
Manage firewall probably should not be hard coded to port 22

### ISSUE THREAD

**Issue by bschonec** [ghoneycutt/puppet-module-ssh#141]
I'm thinking that the following should be tied to $sshd_config_port:

  if $manage_firewall == true {
    firewall { '22 open port 22 for SSH':
      action => 'accept',
      dport  => 22,
      proto  => 'tcp',
    }
  }

If you change the listening port of the ssh daemon, you'd want the firewall (if enabled) to update to the correct port.

**Comment by ghoneycutt** [comment]
Yes, this should use $sshd_config_port in both the title as well as the dport attribute. Could you send a PR?

**Comment by bschonec** [comment]
Also, since SSHD can listen on multiple ports, I'll look at how to make that happen as well.

**Comment by Phil-Friderici** [comment]
Closing as in v4 the firewall functionality got removed.
